xbmc/ContextMenuManager.h

```diff
@@ -67,7 +69,12 @@ class CContextMenuManager
 
 namespace CONTEXTMENU
 {
-  /*!
+/*!
+ * Checks whether any context menu items are available for a file item.
+ * */
+bool HasAnyMenuItemsFor(const std::shared_ptr<CFileItem>& fileItem);
```

Comment:
maybe rename to `ItemHasAnyMenuItems`

Reply:
Not really. We use verb+noun usually for naming methods, and the For is something we also already use, especially in this source file here. I just adapted to the style of the existing class.